microsoftgraph/microsoft-graph-explorer-v4
# Fix: hanging run query button

## Overview

This happens when making anonymous requests.
When a user makes a request and it fails to leave the client (Failed to fetch), the button doesn't stop spinning.

This fixes #1443 

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

The errors encountered when trying to run the queries were not being handled

## Testing Instructions

* Open the link: https://jolly-sand-0ac78c710-1445.centralus.azurestaticapps.net , hit run query
* Notice that the spinner button doesn't spin indefinitely
